assignments/arrays.js

```diff
@@ -63,24 +62,45 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for (var i = 0; i < inventory.length; i++) {
+  //console.log(inventory[i].id === 32)
+  if (inventory[i].id === 33) {
+    //console.log(inventory.car-make)
+    console.log(inventory[i].car_year, inventory[i].car_make, inventory[i].car_model)
+  }
+  //inventory.id  '32'
+}
+console.log(`Car 33 is a` + inventory[32] );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+for (var i = 0; i < inventory.length; i++) {
+  //console.log(inventory[i].id === 32)
+  if (inventory[i].id === 50) {
+    console.log(inventory[i].car_make, inventory[i].car_year);
+  }
+  //inventory.id  '32'
+}
 let lastCar = 0;
-console.log();
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (var i = 0; i < inventory.length; i++) {
+  carModels.push(inventory[i].car_model);
+  console.log(carModels.sort());
```

Comment:
try putting your console log, outside the for loops, so it does not sort every time, and does it only at the end of the loop. I suggest putting it on line 96.